advanced/Templates/pihole-FTL.service

```diff
@@ -26,13 +26,16 @@ start() {
   if is_running; then
     echo "pihole-FTL is already running"
   else
-    touch /var/log/pihole-FTL.log /run/pihole-FTL.pid /run/pihole-FTL.port /var/log/pihole.log
+    touch /var/log/pihole-FTL.log /var/log/pihole.log
```

Comment:
I see two problems with using `install` for this: 

1. We do not want to overwrite the files if they already exist. So running a 
```
install -o pihole -g pihole -m 664 /dev/null "/etc/pihole/dhcp.leases"
```
seems to be no a solution and each file would need an `if` wrapper around them.

2. We do want to change the ownership even if the file already exists as users frequently try to run `pihole-FTL` as `root` and we need to fix all permissions later to allow them (re-)starting `pihole-FTL` later again as user `pihole`.

Reply:
The first issue is solved by checking if the file exists, like the guard at https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh#L1241 

The second issue is something that would not be solved with `install`. 

Can you add a comment that `touch` is being used specifically because the files may already exist and we are just interested in making sure the ownership is correct?

Reply:
Pulling my blocker, this is the sysvinit service file, and my changes would make it too heavy.